samples/bootloader/CMakeLists.txt

```diff
@@ -6,7 +6,29 @@
 
 cmake_minimum_required(VERSION 3.8.2)
 
+
 include($ENV{ZEPHYR_BASE}/cmake/app/boilerplate.cmake NO_POLICY_SCOPE)
 project(bootloader)
 
-target_sources(app PRIVATE src/main.c)
+# This is for allowing the bootloader sample as a separate image in a
+# multi-image build context. It also allows it to be built as a sample
+# if you want to just build the bootloader. ${IMAGE} will be evaluated
+# to empty string(aka. 0) and be the prefix string in a multi-image
+# build.
+
+if(IMAGE)
```

Comment:
There is typically a space after the `if` 

`if (IMAGE`

Reply:
thought that was C file convention but will fix it.